app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -311,7 +311,9 @@ class Overview extends React.Component {
           }
           style={{ marginTop: 200 }}
         >
-          {(transformationMappings.length > 0 || transformationPlans.length > 0 || archivedTransformationPlans) && (
+          {(transformationMappings.length > 0 ||
```

Comment:
Weeeeeeeeell technically so a length of 0 will be false and anything greater than 0 will be true, the` > 0` feels like overkill 🤔